features/progress.yml

```
@@ -1,9 +1,11 @@
-draft_date: 2024-06-05
 name: <progress>
-description: The `<progress>` element TODO.
+description: The `<progress>` element displays an indicator showing the completion progress of a task, typically displayed as a progress bar.
 spec: https://html.spec.whatwg.org/multipage/form-elements.html#the-progress-element
-group: forms
+group:
+  - forms
+  - html-elements
 caniuse: progress
+compute_from: html.elements.progress
```

Sorry, I messed up the suggestion here, though I'm not sure why no error was thrown on the extra key on the top-level object.

```suggestion
status:
  compute_from: html.elements.progress
```